### PR TITLE
Web Dependency Refactor

### PR DESCRIPTION
**Pull request by axelstudios**
Webpack 5 upgrade, which required updating nearly everything else

### CODE REVIEW

**Comment by TShapinsky** [comment]
Any idea what's causing this error during build in github CI?
```
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'babel-preset-minify' imported from /srv/alfalfa/babel-virtual-resolve-base.js
    at new NodeError (/srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:2552:5)
    at packageResolve (/srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3208:9)
    at moduleResolve (/srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3242:18)
    at defaultResolve (/srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3281:13)
    at /srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:3304:14
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:63:103)
    at _next (/srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:65:194)
    at /srv/alfalfa/node_modules/@babel/core/lib/vendor/import-meta-resolve.js:65:364
    at new Promise (<anonymous>) {
  code: 'ERR_MODULE_NOT_FOUND'
}
ERROR: "build-server" exited with 1.
```

**Comment by axelstudios** [comment]
> Any idea what's causing this error during build in github CI?

Hmm, I don't (can't reproduce locally, and we no longer use `babel-preset-minify` (that text doesn't even appear in the repo).  Might be a CI cache issue with node_modules

File: alfalfa_web/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ RUN apt-get update && apt-get install -y \
     ca-certificates \
     git \
     vim \
-    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /var/lib/apt/lists/*
 
 RUN cd /srv
```
Comment (TShapinsky) [inline alfalfa_web/Dockerfile:9]: I think we can also remove the `RUN cd /srv` command as we're following that with a `WORKDIR` command so it literally does nothing

**Review by TShapinsky — APPROVED** [review]
This all looks good to me. Thanks Alex!

**Review by TShapinsky — APPROVED** [review]
Tests passing, looks good to me

**Pull request merged**